### PR TITLE
Add nightly task to build and run tests with latest webrender.

### PR DESCRIPTION
**Pull request by jdm**
Like the nightly Rust compiler upgrade task, this will help us learn sooner when Webrender introduces breaking API changes or causes regressions in our testsuite.

### CODE REVIEW

**Review by Manishearth — APPROVED** [review]
r+

File: etc/taskcluster/decision_task.py [hunk 1]
```diff
@@ -529,6 +530,10 @@ def windows_nightly():
     )
 
 
+def webrender_nightly():
+    return linux_wpt_common(total_chunks=4, layout_2020=False, webrender_nightly=True)
```
Comment (SimonSapin) [inline etc/taskcluster/decision_task.py]: Nit: I’d prefer to keep the calls to `linux_wpt_common` closer to each other, so we see them all when changing for example the number of chunks.

**Comment by atouchet** [comment]
Can this be used to submit a pull request like #23766 suggests?

**Review by SimonSapin — CHANGES_REQUESTED** [review]
I get the desire for something like this PR, but I’m not a fan of:

* Adding any more daily tasks that are expected to sometimes fail without a `servo/servo` PR breaking them
* Adding a known source of intermittent failures (WPT) to daily tasks

… as long as the only feedback we have from them is IRC notifications to a few people.

I tend to take care of compiler updates, but I’m not personally planning to do the same for WebRender. Yet with this PR I’ll get pinged *every day* when WebRender’s master doesn’t build in Servo or has any impact on WPT, until Servo is updated.

**Comment by SimonSapin** [comment]
Possible mitigation: replace this:

https://github.com/servo/servo/blob/a020522f37241001b01ee77dab562d50b6e7a924/etc/taskcluster/decision_task.py#L128

With some per-task configuration where someone is pinged if they personally opt-in.

**Comment by jdm** [comment]
I'll make the pinging configurable like you suggest.

**Comment by jdm** [comment]
@atouchet That's more effort than I'm willing to spend on this task right now, given the complexities involved in https://github.com/servo/servo/blob/master/etc/ci/update-wpt-checkout.

**Comment by mrobinson** [comment]
Going to close this PR since it's against the old taskcluster config.

**Pull request closed without merging**